src/shogun/multiclass/MulticlassSVM.cpp

```diff
@@ -45,6 +45,8 @@ bool MulticlassSVM::create_multiclass_svm(int32_t num_classes)
 	if (num_classes>0)
 	{
 		m_machines.clear();
+		for (auto i: Range(0, m_multiclass_strategy->get_num_machines()))
+			m_machines.push_back(nullptr);
```

Comment:
@vigsterkr this a bit of a nonsense, but this is what it was before and now it works...